GoogleSignIn/Sources/Public/GoogleSignIn/GIDGoogleUser.h

```diff
@@ -53,6 +57,18 @@ NS_ASSUME_NONNULL_BEGIN
 /// see https://developers.google.com/identity/sign-in/ios/backend-auth.
 @property(nonatomic, readonly, nullable) GIDToken *idToken;
 
+/// Gets a new authorizer for `GTLService`, `GTMSessionFetcher`, or `GTMHTTPFetcher`.
```

Comment:
Update the doc comment to reflect that this will always return the same instance.

Reply:
This will change when the `authState` is updated by either refreshing tokens or adding new scopes.